src/procedures/procedures.11tydata.yaml

```diff
@@ -340,3 +340,77 @@ procedureRemoveCM4:
           <li class="info">This requires quite a bit of strength. If it does not work, pull harder.</li>
           <li class="warning">Do not use a screw driver as a lever. It might damage the board.</li>
         </ul>
+procedureEnableMultiProtocol:
+  title: Enable Multiprotocol support
+  steps:
+    - title: Enable multiprotocol support
+      image: yellow-multiprotocol-01.png
+      content: |
+        <ul>
+          <li class="warning">Multiprotocol is an experimental feature. It may not work for you.</li>
+          <li class="info">When multiprotocol support is enabled, the Home&nbsp;Assistant Yellow's IEEE&nbsp;802.15.4 radio can be used for both Zigbee and Thread (used by Matter) at the same time. The two networks will be running on the same IEEE&nbsp;802.15.4 channel.</li>
+          <li>If the radio is already used by the ZHA Zigbee integration, ZHA will be automatically reconfigured to use the multiprotocol firmware. Your Zigbee network will continue to work.</li>
+        </ul>
+        <ol>
+          <li>Go to <a class="my" href="https://my.home-assistant.io/redirect/hardware/" target="_blank"><b>Settings</b> > <b>System</b> > <b>Hardware</b></a>.</li>
+          <li>Under <b>Home Assistant Yellow</b>, select <b>Configure</b>.</li>
+          <li>Select the <b>Enable multiprotocol support</b> checkbox and select <b>Submit</b>.</li>
+        </ol>
+    - title: Wait
+      image: skyconnect-multiprotocol-02.png
+      content: |
+        <ul>
+          <li>Wait for the installation of the <b>Silicon Labs Multiprotocol</b> add-on to complete.</li>
+        </ul>
+    - title: Installation complete
+      image: skyconnect-multiprotocol-03.png
+      content: |
+        <ul>
+          <li>Once the installation is complete, select <b>Finish</b>.</li>
+        </ul>
+    - title: Locate the Silicon Labs Multiprotocol add-on
+      image: yellow-multiprotocol-04.png
+      content: |
+        <ul>
+          <li>Go to <a class="my" href=" https://my.home-assistant.io/redirect/supervisor_addon/?addon=core_silabs_multiprotocol" target="_blank"><b>Settings</b> > <b>Add-ons</b></a>.<ul>
+          <li>You should now see the <b>Silicon Labs Multiprotocol</b> add-on.</li></ul>
+        </ul>
```

Comment:
I think I would remove that step, we already mention above that it gets installed above. The add-on is largely managed by the feature, and shouldn't be of concern for the user. 

We also discussed to maybe hide (to some degree) this type of system installed add-ons, to avoid accidental interference.